Sync_app/moysklad/moysklad_class_lib.py

```diff
@@ -518,3 +531,66 @@ def get_retail_demand_by_period(
                         quantity=goods[position.good.good_id].quantity + int(position.quantity),
                     )
         return list(goods.values())
+
+    def get_retail_sales_return_by_period(
+        self,
+        start_period: Optional[datetime.date],
+        end_period: Optional[datetime.date],
```

Comment:
Можно сделать так и не передавать None когда ты не используешь параметр
```suggestion
        start_period: Optional[datetime.date] = None,
        end_period: Optional[datetime.date] = None,
```